### DISCUSSION TITLE
Merge for bevy 0.10 ?

### DISCUSSION THREAD

**Discussion by Rayoule** [abnormalbrain/bevy_particle_systems#41] [Q&A] [answered]
Hey !
I wanted to know, when the compatibility for bevy 0.10 will be available ?
Thanks a lot :)

**Comment by abnormalbrain** [comment, marked as answer]
Hi @Rayoule, apologies for the delay, I merged 0.10 supported and released bevy_particle_systems v0.9 last night, so it should be available now! 😅

**Reply by Rayoule** [reply]
Thanks !
